.dir-locals.el

```diff
@@ -0,0 +1,12 @@
+;;; Directory Local Variables
+;;; For more information see (info "(emacs) Directory Variables")
```

Comment:
Like as I always said 😛

We need an EditorConfig, that too in a separate commit, please.